xml/SLES-JeOS-12SP4-Quickstart.xml

```diff
@@ -225,51 +226,49 @@
      </itemizedlist>
    </sect3>
 
-   <sect3 xml:id="sec-list-preinstalled-packages">
-    <title>List of Preinstalled Packages</title>
-
-    <para>The kiwi configuration files used to create the official SUSE Linux Enterprise
-     Server JeOS images are available from the openSUSE Build Service at <link
-     xlink:href="https://build.opensuse.org/">https://build.opensuse.org/</link>. For
-     example you can check the packages list for SUSE Linux Enterprise Server 15
-     JeOS:</para>
-    <para>
-    <link xlink:href="https://build.opensuse.org/package/view_file/SUSE:SLE-15:GA/kiwi-templates-SLES15-JeOS/JeOS.kiwi?expand=1"/></para>
-
-    <para>To get a detailed list of the specific host tools packages, check:</para>
-     <itemizedlist>
-      <listitem>
-       <para><package>open-vm-tools</package> for VMWare</para>
-      </listitem>
-      <listitem>
-       <para><package>hyper-v</package> package for Hyper-V image</para>
-      </listitem>
-     </itemizedlist>
-   </sect3>
-
-   <sect3 xml:id="sec-jeos-to-full-sles">
-    <title>From JeOS to a Full SUSE Linux Enterprise Server System</title>
-
-    <para>You can make a JeOS image closer to a full SUSE Linux Enterprise Server system by
-     manually installing packages and patterns. For example, if you want to add a
-     Graphical Desktop Environnement, you can install either the GNOME or LXDE desktop
-     environment with the command <command>zypper in</command>.</para>
+   <sect3 xml:id="sec-list-base-packages">
+    <title>List of Base Packages</title>
```

Comment:
This uses `Base Packages` instead of `Preinstalled Packages` without explaining what it means.